test/e2e/storage/utils/create.go

```diff
@@ -355,8 +359,14 @@ func patchItemRecursively(f *framework.Framework, driverNamespace *v1.Namespace,
 		}
 	case *v1.Service:
 		PatchNamespace(f, driverNamespace, &item.ObjectMeta.Namespace)
-	case *appsv1.StatefulSet:
+	case *appsv1.Deployment:
```

Comment:
I think csi hostpath uses StatefulSet, so we may need to add Deployment as an additional case.

Reply:
Yeah, I figured that out when the other tests started failing... :-/